addons/calendar/models/calendar_event.py

```diff
@@ -958,11 +958,26 @@ def get_display_time_tz(self, tz=False):
             self = self.with_context(tz=tz)
         return self._get_display_time(self.start, self.stop, self.duration, self.allday)
 
+<<<<<<< HEAD
     def _get_ics_file(self):
         """ Returns iCalendar file for the event invitation.
             :returns a dict of .ics file content for each meeting
         """
         result = {}
+=======
+        # Add commands to create attendees from partners (if present) if no attendee command
+        # is already given (coming from Google event for example).
+        vals_list = [
+            dict(vals, attendee_ids=self._attendees_values(vals['partner_ids']))
+            if 'partner_ids' in vals and 'attendee_ids' not in vals
+            else vals
+            for vals in vals_list
+        ]
+        recurrence_fields = self._get_recurrent_fields()
+        recurring_vals = [vals for vals in vals_list if vals.get('recurrency')]
+        other_vals = [vals for vals in vals_list if not vals.get('recurrency')]
+        events = super().create(other_vals)
+>>>>>>> ee65e618cb0... temp
```

Comment:
@remybar 
The order of methods was modified to clean the code in 14.3 or 14.4. This explains the conflict. 
In 15.0, we have
```py
        # Automatically add the current partner when creating an event if there is none (happens when we quickcreate an event)
        default_partners_ids = defaults.get('partner_ids') or ([(4, self.env.user.partner_id.id)])
        vals_list = [
            dict(vals, attendee_ids=self._attendees_values(vals.get('partner_ids', default_partners_ids))) if not vals.get('attendee_ids') else vals
            for vals in vals_list
        ]
```
You propose this one that will not add the current partner during the quick create. That's an issue.
```py
        vals_list = [
            dict(vals, attendee_ids=self._attendees_values(vals['partner_ids']))
            if 'partner_ids' in vals and 'attendee_ids' not in vals
            else vals
            for vals in vals_list
        ]
```
I think the approach with the default value will take into account your issue. What do you think ?